### PR TITLE
Add info about comments in test list

### LINKED ISSUE #669: Support lines commented-out in TestList files
I use Test List files quite a bit with NUnit Console --TestList option.  Sometimes I would like to be able to not run a specific test in the list, but still keep it in the list to re-enable it later.  It would be great if I could comment-out a line in the list when that is necessary.  There isn't a lot of info on the test list files, so I wasn't sure if is possible to do this now or not, but it would be a nice feature to have if it isn't currently supported.

### PR DESCRIPTION
**Pull request by CharliePoole**
Resolves #669.

### CODE REVIEW

**Comment by SeanKilleen** [comment]
Hi @CharliePoole -- I added a note to your PR description to indicate it will resolve #669 when merged. 

If the PR is ready for review, let me know. If you're still working on it, feel free to add "WIP" to the PR title, which will block it for merge until you're ready for review.

File: docs/articles/nunit/running-tests/Console-Command-Line.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Without the V2 driver, only version 3.0 and higher tests may be run.
 | ------ | ----------- |
 |`@FILE` | Specifies the name (or path) of a FILE containing additional command-line arguments to be interpolated at the point where the @FILE expression appears. Each line in the file represents a separate command-line argument.|
 |`--test=FULLNAMES` | Comma-separated list of FULLNAMES of tests to run or explore. This option may be repeated. Note that this option is retained for backward compatibility. The --where option can now be used instead. |
-|`--testlist=FILE` | The name (or path) of a FILE containing a list of tests to run or explore, one per line. |
+|`--testlist=FILE` | The name (or path) of a FILE containing a list of tests to run or explore, one per line. May also include comment lines, indicated by '#' in the first column.|
```
Comment (SeanKilleen) [inline docs/articles/nunit/running-tests/Console-Command-Line.md:37]:
```suggestion
|`--testlist=FILE` | The name (or path) of a FILE containing a list of tests to run or explore, one per line. May also include comment lines, indicated by `#` in the first column.|
```

**Comment by CharliePoole** [comment]
I intended it to be committed once you approved it. I'm not authorized to commit it myself, of course.

File: docs/articles/nunit/running-tests/Console-Command-Line.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Without the V2 driver, only version 3.0 and higher tests may be run.
 | ------ | ----------- |
 |`@FILE` | Specifies the name (or path) of a FILE containing additional command-line arguments to be interpolated at the point where the @FILE expression appears. Each line in the file represents a separate command-line argument.|
 |`--test=FULLNAMES` | Comma-separated list of FULLNAMES of tests to run or explore. This option may be repeated. Note that this option is retained for backward compatibility. The --where option can now be used instead. |
-|`--testlist=FILE` | The name (or path) of a FILE containing a list of tests to run or explore, one per line. |
+|`--testlist=FILE` | The name (or path) of a FILE containing a list of tests to run or explore, one per line. May also include comment lines, indicated by `#` in the first column.|
```
Comment (SeanKilleen) [inline docs/articles/nunit/running-tests/Console-Command-Line.md:37]:
```suggestion
|`--testlist=FILE` | The name (or path) of a FILE containing a list of tests to run or explore, one per line. May also include comment lines, indicated by `#` in the first column. |
```

Adding a space prior to the pipe. Very very minor. I'll commit this one myself and then merge.

**Review by SeanKilleen — APPROVED** [review]

**Comment by SeanKilleen** [comment]
Thanks for this, @CharliePoole !

**Pull request merged**